features/step_definitions/hangout_steps.rb

```diff
@@ -182,4 +185,7 @@
   page.should have_field('hangout_url', with: @hangout_url)
 end
 
-
+Then(/^a separate event instance is not created$/) do
+  expect(EventInstance.where("created_at >= ?", Time.now.beginning_of_day).size)
```

Comment:
I think the date coded at the beginning of the cuke test will persist to this step. Let me check using byebug.